internal/pb/export/export.proto

```diff
@@ -67,22 +75,39 @@ message TemporaryExposureKey {
   optional bytes key_data = 1;
 
   // Varying risks associated with exposure depending on type of verification
-  optional int32 transmission_risk_level = 2;
+  // Ignored by the v1.5 client API when report_type is set.
+  optional int32 transmission_risk_level = 2 [deprecated = true];
```

Comment:
Is what we refer to as "v1.5" a notion that external devs will have access to? On iOS side this'll be an os version?

Reply:
I believe we've been messaging it that way and Apple will do the same.

Reply:
I.e. API version 1.5